NAS2D/Mixer/Mixer.h

```diff
@@ -56,15 +56,9 @@ namespace NAS2D
 		 *
 		 * \param music Reference to a Music Resource.
 		 * \param loops Repeat count. -1 for continuous loop.
-		 * \param time Length of the fade in milliseconds. Default is 500.
```

Comment:
I'm a little bit concerned that documenting some but not all parameters could trigger a warning/error from some of the documentation checks. Would be nice to get all CI builds to run and see if they complain.

Reply:
Would removing the entire comment block be better?

Reply:
Possibly. I feel a little on the fence on this one.

I think maybe the parameter should be renamed to something like `fadeTime`. Then I'd probably be comfortable removing the documentation.